### PR TITLE
CDK: show main-menu items if AWS panel is hidden; search for projects only if CDK is enabled

### PR DESCRIPTION
**Pull request by justinmk3**
## Problem

1. If CDK view is hidden, it's wasteful to search for CDK projects.
1. Any view can be shown/hidden. But the Toolkit "standard" menu items (such as help docs, feedback, etc.) are defined in the AWS view. 

## Solution

1. Skip search if CDK view is not visible; refresh() when set to visible.
1. If the AWS view is hidden, show the "standard" menu items in non-AWS view menus.
    - ref e132be025e90 #1632

aws-cdk-panel-menus

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -801,27 +801,27 @@
...
                 {
                     "command": "aws.help",
-                    "when": "view == aws.explorer",
+                    "when": "view == aws.explorer || !aws.explorer.visible",
                     "group": "y_externalLinks@1"
                 },
                 {
                     "command": "aws.github",
-                    "when": "view == aws.explorer",
+                    "when": "view == aws.explorer || !aws.explorer.visible",
```
Comment (kiiadi) [inline package.json:809]: how does this scale as we add other "top-level" groups?

Comment (justinmk3) [reply package.json:809]: The idea is that, at least currently, the AWS panel is the main panel. If it's hidden, then all panels show the "general" items.

Comment (awschristou) [reply package.json:809]: Does this mean the AWS Toolkit menu items show up in the menus of non-AWS Toolkit panels? We had a similar bug earlier on like that.

Comment (justinmk3) [reply package.json]: Fixed in https://github.com/aws/aws-toolkit-vscode/pull/1679/commits/57356fce4905a6b7adf88b15d133e72a02f3569b

**Review by bryceitoc9 — APPROVED** [review]

**Pull request merged**